src/local-chain-sync.ts

```diff
@@ -34,8 +34,14 @@ class Database {
     }
 }
 
+const replacer = (_key: any, block: Block) => {
+    // Avoids error when serializing BigInt
+    delete block['transactions'];
+    return block;
```

Comment:
Feels a bit destructive and looses a bit the value of printing the block to begin with? What about stringifying BigInts instead? Something like:

```ts
const replacer = (_key: any, value: any) => value instanceof BigInt ? value.toString() : value;
```

Reply:
Good call. Works great now, but I had to make a slight adjustment (using `typeof`)